modules/installation-special-config-kmod.adoc

```diff
@@ -0,0 +1,306 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+
+[id="installation-user-infra-generate-k8s-manifest-ignition-rhcos_{context}"]
+= Adding kernel modules to nodes
+
+For most common hardware, the Linux kernel includes the device driver
+modules needed to use that hardware when the computer starts up. For
+some hardware, however, modules are not available in Linux. Therefore, you must
+find a way to provide those modules to each host computer. This
+procedure describes how to do that for nodes in an {product-title} cluster.
+
+When a kernel module is first deployed following these instructions,
+the module is made available for the current kernel. If a new kernel
+is installed, the kmods-via-containers software will rebuild and deploy
+the module so a compatible version of that module is available with the
+new kernel.
+
+The way that this feature is able to keep the module up to date on each
+node is by:
+
+* Adding a systemd service to each node that starts at boot time to detect
+if a new kernel has been installed and
+* If a new kernel is detected, the
+service rebuilds the module and installs it to the kernel
+
+For information on the software needed for this procedure, see the
+link:https://github.com/dustymabe/kmods-via-containers[kmods-via-containers] github site.
+
+[IMPORTANT]
+====
+In this procedure, the software needed to build your kernel modules is
+deployed in a RHEL container. Keep in mind that modules are rebuilt
+automatically on each node when that node gets a new kernel. For that
+reason, each node needs access to a yum repository that contains the
+kernel and related packages needed to rebuild the module. That content
+is best provided with a valid RHEL subscription.
+====
+
+== Build and test the kernel module container
+Gather the kernel module’s source code, the KVC framework, and the
+kmod-via-containers software. Then build and test the module. To do
+that on a RHEL 8 system, do the following:
+
+Procedure
+
+. Get a RHEL 8 system, then register and subscribe it:
++
+----
+# subscription-manager register
+Username: yourname
+Password: ***************
+# subscription-manager subscribe --auto
+----
+
+. Install software needed to build the software and container:
++
+----
+# yum install podman make git -y
+----
+
+. Clone the kmod-via-containers repository:
++
+----
+$ mkdir kmods; cd kmods
+$ git clone https://github.com/dustymabe/kmods-via-containers.git
+----
+
+. Install a KVC framework instance on your build host to test the module.
```

Comment:
change to "...on your RHEL 8 build host..."  ?

Reply:
Done.